daprdocs/content/en/reference/environment/_index.md

```diff
@@ -8,12 +8,9 @@ weight: 300
 
 The following table lists the environment variables used by the Dapr runtime, CLI, or from within your application:
 
-| Environment Variable             | Used By          | Description |
-|----------------------------------|------------------|-------------|
-| DAPR_HTTP_PORT                   | Your application | The HTTP port that Dapr is listening on. Your application should use this variable to connect to Dapr instead of hardcoding the port value. Injected by the `dapr-sidecar-injector` into all the containers in the pod.
-| DAPR_GRPC_PORT                   | Your application | The gRPC port that Dapr is listening on. Your application should use this variable to connect to Dapr instead of hardcoding the port value. Injected by the `dapr-sidecar-injector` into all the containers in the pod.
-| DAPR_TOKEN_API                   | Your application | The token used for Dapr API authentication for requests from the application. Read [enable API token authentication in Dapr]({{< ref api-token >}}) for more information.
-| APP_TOKEN_API                    | Your application | The token used by the app to authenticate requests from Dapr. Read [authenticate requests from Dapr using token authentication]({{< ref app-api-token >}}) for more information.
-| DAPR_PLACEMENT_HOST              |Your application | The address for the Dapr Placement service. Only needed in self-hosted mode if you run your app (which creates actors) and you want to tell the app the location of the Placement service. This is never needed outside of local machine development.
-| DAPR_NETWORK                     | Dapr CLI         | Optionally used by the Dapr CLI to specify the Docker network on which to deploy the Dapr runtime.
-| NAMESPACE                        | Dapr runtime     | Used to specify a component's [namespace in self-hosted mode]({{< ref component-scopes >}})
+| Environment Variable | Used By          | Description                                                                                                                                                                                                             |
+| -------------------- | ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| DAPR_HTTP_PORT       | Your application | The HTTP port that Dapr is listening on. Your application should use this variable to connect to Dapr instead of hardcoding the port value. Injected by the `dapr-sidecar-injector` into all the containers in the pod. |
```

Comment:
Update these to have better details

| DAPR_HTTP_PORT | Your application | The HTTP port that Dapr sidecar is listening on. Your application should use this variable to connect to Dapr sidecar instead of hardcoding the port value. Set by the Dapr CLI run command for self hosted or injected by the `dapr-sidecar-injector` into all the containers in the pod. |

Reply:
Finally this PR also need to update the Env variable column in the CLI docs. These ones
1) https://docs.dapr.io/reference/cli/dapr-init/ - Remove DAPR_NETWORK
2) https://docs.dapr.io/reference/cli/dapr-run/ - Add DAPR_GRPC_PORT and DAPR_HTTP_PORT env var. Remove DAPR_NETWORK
3) Same file CLI run,  https://docs.dapr.io/reference/cli/dapr-run/ -  The --metrics-port and DAPR_METRICS_PORT  parameter is missing entirely. Add description "The port that Dapr sends its metrics information to"
4) https://docs.dapr.io/reference/cli/dapr-uninstall/ - Remove DAPR_NETWORK

Reply:
With the above changes this should now be correct with Dapr and CLI. Effectively there are either env that are injected into the app context from CLI commands or from the hosting env (such as K8s). Or there are env variable that are read when Dapr starts to affects its runtime, notable authentication tokens and namespacing in self hosted mode.

Reply:
@msfussell 

> https://docs.dapr.io/reference/cli/dapr-run/ - Add DAPR_GRPC_PORT and DAPR_HTTP_PORT env var. Remove DAPR_NETWORK

Do we need to add APP_ID, APP_PORT?

Reply:
I fixed it